src/Microsoft.Identity.Client/Exceptions/CoreErrorMessages.cs

```diff
@@ -165,5 +165,8 @@ public static string CustomWebUiRedirectUriWasNotMatchedToProperUri(string expec
         }
 
         public const string CustomWebUiAuthorizationCodeFailed = "CustomWebUi AcquireAuthorizationCode failed";
+
+        public const string TokenCacheJsonSerializerFailedParse = "MSAL V3 Deserialization failed to parse the cache contents.  Is this possibly an earlier format needed for DeserializeMsalV2?";
```

Comment:
>TokenCacheJsonSerializerFailedParse [](start = 28, length = 35)

suggestion: Deserialization failed to parse the content. Your data might be of a different format (See: DeserializeMsalV3, DeserializeMsalV2, DeserializeAdalV3).